app/src/androidTest/java/com/example/sharingang/UserProfileFragmentTest.kt

```diff
@@ -64,6 +65,20 @@ class UserProfileFragmentTest {
         onView(withId(R.id.btnApply)).check(matches(not(isDisplayed())))
     }
 
+    @Test
+    fun reportUserIsDisplayedAndSendsReport() {
+        navigate_to(R.id.userProfileFragment)
+        onView(withId(R.id.btn_report)).check(matches(isDisplayed()))
+        onView(withId(R.id.btn_report)).perform(click())
+        val device: UiDevice = UiDevice.getInstance(getInstrumentation())
```

Comment:
```suggestion
        val device = UiDevice.getInstance(getInstrumentation())
```
Seems better to me without the explicit annotation, given the `UIDevice` appearing on the right as well. If this annotation is necessary, than ignore this